### ISSUE TITLE
Focus in table resets after 5 seconds

### ISSUE THREAD

**Issue by jgiardino** [bf2fc6cc711aee1a0c2a/kas-ui#235]
For keyboard accessibility, I should be able to tab through the UI. Currently, the table creates a bit of a focus trap.
When I tab through the elements in the table, I see focus shift from link to kebab, from row to row as expected. Until after about 5 seconds, and the table refreshes. At this point in time, my focus is reset to the name link in the first table row. 

This breaks at least one of the WCAG criteria for not creating a focus trap, but I think there's also one about forcing users to act within a limited time range to complete an action.
